lib/ts/components/componentOverride/componentOverrideContext.ts

```diff
@@ -1,8 +1,10 @@
 import React from "react";
 import { ComponentOverride } from "./componentOverride";
 
-type ContextType<T> = {
-    [K in keyof T]?: ComponentOverride<any>;
-};
+type ContextType<T> =
+    | {
+          [K in keyof T]?: ComponentOverride<any>;
+      }
+    | "IS_DEFAULT";
```

Comment:
Why use `IS_DEFAULT` string instead of `undefined` or `null` for instance?

Reply:
Just to be more explicit.. Could have used `undefined` or `null` as well.. But they we would not be able to differentiate if the user has provided `undefined` as opposed to it being our mistake where we forgot to add a parent provider.

Reply:
Right, the mistake might also be related to passing invalid value in normalize config as well. Let's go this way.